test-apps/display-test-app/src/frontend/ViewPicker.ts

```diff
@@ -33,7 +34,18 @@ export class ViewList extends SortedArray<ViewSpec> {
   public async getView(id: Id64String, iModel: IModelConnection): Promise<ViewState> {
     let view = this._views.get(id);
     if (undefined === view) {
-      view = await iModel.views.load(id);
+      if (Id64.isInvalid(id)) {
+        view = this.manufactureSpatialView(iModel);
+      } else {

```

Comment:
seems like this test should be in `views.load`

Reply:
I don't think views.load should be silently generating a fake spatial view.

Reply:
no, it should throw if it gets an invalid id. Then you don't need a separate test.

Reply:
The check on invalid Id here is to handle opening a blank connection - I just moved the code from elsewhere in this file.

Reply:
my point remains. What should view.load do if it gets an invalid Id other than throw (it must already, somehow)?

Reply:
Yeah, your edits and my replies are getting crossed. I don't think views.load should be changed for the convenience of display-test-app but it should throw IModelStatus.InvalidId if input is invalid to avoid useless backend call and produce clearer exception.

Reply:
yes, and a side benefit of that improvement will be that you don't need a separate test here.

Reply:
done.

Reply:
I will need you to re-approve.